atlp-rwanda/e-comm-team-techsmith-fn
# #185467879: Buyer should be able delete single item and delete All items in wishlist

## What does this PR do?
This PR implement Deleting single item and delete all item in wishlist

## Description of Task to be completed?
- Delete both single item and All items in wishlist 

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-buyer-should-delete-single-item-and-all-items-in-wishlist#185467879`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?

- [#185467879](https://www.pivotaltracker.com/story/show/185467879)
## Screenshots (if appropriate)
